src/arithmetic/func_desc.c

```
@@ -44,19 +127,93 @@ AR_FuncDesc *AR_FuncDescNew
 	ASSERT(res == 1);
 }
 
+// forward declaration
+SIValue AR_UDF (SIValue *argv, int argc, void *private_data) ;
+
+// register a new UDF function
+void AR_FuncRegisterUDF
+(
+	char *name  // full name (lib.func)
+) {
+	ASSERT (name != NULL) ;
```

Cppcheck (2.18.0)

[error] Null pointer dereference

(nullPointer)